src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -493,6 +493,24 @@ internal bool IsNetStandardFeatureFlagEnabled()
             return value == "1" || value == "true";
         }
 
+        internal IDictionary<string, string> InitializeHeaderTags(IDictionary<string, string> configurationDictionary)
+        {
+            IDictionary<string, string> headerTags = new Dictionary<string, string>();
```

Comment:
Just wondering why explicit `IDictionary<>` interface here rather than var? I realise the return type is `IDictionary<>` just wondering if there's some performance aspect I'm not aware of 🙂

Reply:
I really don't know why I did that, I'll update this to use `var` so there's no confusion

Reply:
Done in b3f9c8c9975f0bbf1ada23a363a69edf4e77ac8d